projects/plugins/jetpack/extensions/blocks/business-hours/editor.scss

```diff
@@ -36,7 +51,7 @@
 		.business-hours__day {
 			width: 44%;
 			display: flex;
-			align-items: baseline;
+			align-items: end;
```

Comment:
This is testing well for me in Chrome and Edge so far, but in Firefox setting to `align-items: end` results in the day text and toggle button being lower down within the element:

![image](https://user-images.githubusercontent.com/14988353/114822327-3cfa6500-9e05-11eb-846b-db85cd4e7811.png)

Swapping this out for `align-items: start` appears to fix it, though, and I think works the same in Chrome, Edge and Safari?

```suggestion
			align-items: start;
```